x-pack/plugin/ent-search/qa/rest/src/yamlRestTest/resources/rest-api-spec/test/entsearch/332_connector_update_filtering.yml

```diff
@@ -41,7 +44,13 @@ setup:
                 advanced_snippet:
                   created_at: "2023-05-25T12:30:00.000Z"
                   updated_at: "2023-05-25T12:30:00.000Z"
-                  value: {}
+                  value:
```

Comment:
Should we add one additional test, which tests that parsing an object to a map with actual values works? I think we only have empty objects like `value: {}` in this integration test. It could also make sense to separate the tests to `Update Connector Filtering with advanced snippet value array`, `Update Connector Filtering with advanced snippet value object`, `Update Connector Filtering with advanced snippet value literal`, but this is just a personal opinion.

Reply:
Let's add it in followup PR, I want to get this merged before next BC